db/migrate/20151103223107_drop_properties.rb

```diff
@@ -0,0 +1,5 @@
+class DropProperties < ActiveRecord::Migration
+  def change
```

Comment:
There's no precedent for doing that in our db/migrations so far, so I don't think we need it.